package.json

```diff
@@ -65,6 +65,7 @@
     "rimraf": "^3.0.2",
     "rollup": "^2.56.2",
     "sinon": "^11.1.2",
+    "source-map": "^0.7.3",
```

Comment:
Ok. Actually, I doubted should I depend on the beta version of the project that not seems to be actively developed for 2 years for now. Even now, npm offers to install version `0.6.1` instead of the last available version (although now I realized that this is due to the fact that such version is specified in `package-lock.js` at the top level)

Reply:
I'm working up that PR for source-map-generator, just in case.

Reply:
actually, that repo seems to be relatively unmaintained.  I'm doing a generator-only fork of source-map to see how it looks.

Reply:
I went aggressive: https://github.com/hildjj/source-map-generator